docs/introduction/sla.md

```diff
@@ -0,0 +1,208 @@
+ * Any other causes outside service provider’s direct control
+
+## Communication, reporting and escalation
+
+### General communication
+
+The following contacts will be generally used for communications related to the
+service in the scope of this SLA:
+
+| Communication channels ||
+--- | ---
+Contact for Customers | servicedesk@csc.fi
+Security incident reporting | (CSC’s Head of Security) security@csc.fi +358 (0) 94 57 2253
+
+### Reporting
+The service availability information is…. The information provided will be limited to service availability and by data security and privacy constraints.
+
+### Escalation and complaints
+
+For escalation and complaints, the defined contact point shall be used, and the
+following rules apply:
+
+1. First contact shall be established, preferably by e-Mail, to Contact for Customers address (See section 8.1) explaining the reason for the complaint with a sensible level of detail and clarity. Please also include, if possible, the following information:
+    * Name of the service
+    * Date and time of the events
+    * Usernames of affected users
+    * Channel to use on following communications (If other is preferred)
+2. CSC Service Desk will contact you back within three working days with information about the incident and which procedures will be adopted. 
+
+## Information security & data protection
+ CSC has approved a security policy and also follows security best practices.
+For CSC's customers, partners and staff there are detailed security guidelines.
+Many items in our security policies and guidelines refer to external compliance
+requirements. CSC also has procedures for risk and security management. For
+more information, please refer to the following pages:
+
+https://www.csc.fi/security
+
+https://www.csc.fi/web/research/rahti-security
+
+The handling of personal information on the Rahti service is described here.
+
+https://www…..
+
+## Additional responsibilities of the Service Provider
+
+Additional responsibilities of the Service Provider are as follow:
+
+ * Adhere to all applicable operational and security policies and procedures defined in CSC’s Security Policy (See section 9) and to other policy documents referenced therein
+ * Use communication channels defined in the agreement
+ * Provide monitoring data to measure fulfillment of agreed service level targets.
+
+## Customer responsibilities
+
+The Customer agrees to follow the General Terms of Use for CSC's Services for
+Science7, User policy and terms of usage for the Rahti container cloud service8
+and CSC’s Security Policy9.
+
+## Review
+
+There will be reviews of the service performance against service level targets
+and of this SLA at planned intervals according to the following rules: 
+
+ * Annual reviews are done internally and based on customer feedback
+ *  Major changes to the service may trigger a review.
+
+## Glossary of terms
```

Comment:
Any necessary terms from the ToU document also used in this document could be listed here as well.

Reply:
Added generic relevant terms from the ToU, and tried to fix the wording across the document to match.

Reply:
Could you also make the usage of emphasis the same across the items in the glossary list? Some items now have emphasis while others do not.

Reply:
Yes. Made